si/orange-canvas.yaml

```diff
@@ -0,0 +1,3536 @@
+            quickmenu/size: false
+    class `WindowSizeGrip`:
+        def `setCorner`:
+            Qt.Corner flag expected: false
+        def `sizeHint`:
+            QMacStyle: false
+document/schemeedit.py:
+    application/vnd.{}-ows-fragment+xml: false
+    class `NoWorkflowError`:
+        def `__init__`:
+            No workflow model is set: false
+    class `SchemeEditWidget`:
+        class `OpenAnchors`:
+            Never: false
+            Always: false
+            OnShift: false
+        def `__init__`:
+            '&Edit': Uredi
+            Widget: Gradnik
+            '&Widget': Gradnik
+            Link: Povezava
+        def `__setupActions`:
+            Clean Up: Počisti
+            cleanup-action: false
+            Shift+A: false
+            Align widgets to a grid (Shift+A): Poravnaj vse gradnike na mrežo (Shift+A)
+            Text: Besedilo
+            new-text-action: false
+            Add a text annotation to the workflow.: V delotok dodaj besedilno oznako.
+            Font Size: Velikost pisave
+            '%ipx': false
+            Arrow: Puščica
+            new-arrow-action: false
+            Add a arrow annotation to the workflow.: V delotok dodaj oznako s puščico.
+            Arrow Color: Barva puščice
+            '#000': false
+            '#C1272D': false
+            '#662D91': false
+            '#1F9CDF': false
+            '#39B54A': false
+            undo-action: false
+            redo-action: false
+            Select all: Izberi vse
+            select-all-action: false
+            Select all items.: Izberi vse elemente.
+            Open: Odpri
+            open-action: false
+            Open selected widget: Odpri izbrane gradnike
+            Remove: Odstrani
+            remove-selected: false
+            Remove selected items: Odstrani izbrane gradnike
+            Ctrl+Backspace: fales
+            Rename: Preimenuj
+            rename-action: false
+            Rename selected widget: Preimenuj izbrane gradnike
+            darwin: false
+            Help: Pomoč
+            help-action: false
+            Show widget help: Pokaži pomoč za gradnik
+            F1: false
+            Enabled: Vključeno
+            link-enable-action: false
+            link-remove-action: false
+            Remove link.: Odstrani povezavo.
+            Insert Widget: Vstavi gradnik
+            node-insert-action: false
+            Insert widget.: Vstavi gradnik.
+            Reset Signals: Ponastavi signale
+            link-reset-action: false
+            Duplicate: Podvoji
+            duplicate-action: false
+            Ctrl+D: false
+            Copy: Kopiraj
+            copy-action: false
+            Ctrl+C: false
+            Paste: Prilepi
+            paste-action: false
+            Ctrl+V: false
+            Window Groups: Skupine oken
+            window-groups-action: false
+            Manage preset widget groups: Upravljaj postavljene skupine gradnikov
```

Comment:
"Upravljaj prednastavljene skupine gradnikov"

Reply:
V bistvu je tako, kot sem prevedel, po mojem bolj prav. Uporabnik si lahko postavi določene skupine gradnikov in jih shrani. Zdaj sem spremenil v "Upravljaj postavitve skupin gradnikov".